test-runner/wasi_test_runner/test_suite_runner.py

```diff
@@ -73,9 +81,12 @@ def _skip_single_test(
 
 
 def _execute_single_test(
-    runtime: RuntimeAdapter, validators: List[Validator], test_path: str
+    runtime: RuntimeAdapter,
+    validators: List[Validator],
+    test_path: str,
+    config_override: Optional[Config],
```

Comment:
given that this eventually is optional anyway, could we perhaps remove the `StubConfigOverride` class and pass around `None` if no override is provided?